pages/_app.jsx

```diff
@@ -3,10 +3,17 @@ import "../styles/globals.scss";
 import {TimelineAnimationWrapper} from "../components/TimelineWrapper";
 import ResizeProvider from "../components/ResizeProvider";
 import {useRouter} from "next/router";
+import {useEffect, useState} from "react";
 
 function MyApp({Component, pageProps}) {
 	const router = useRouter();
-	const author = router.query["authorHost"];
+	const [author, setAuthor] = useState();
```

Comment:
in the race condition, the meta tag is present but not loaded from the Universal Editor